server/services/reportSummary.ts

```diff
@@ -35,8 +45,15 @@ const createIncidentDetails = (
   }
 }
 
-const createUseOfForceDetails = (details = {}) => {
+const createUseOfForceDetails = (
+  details: Partial<UseOfForceDetails> = {},
+  reasonsForUseOfForce: Partial<ReasonsForUseOfForce> = {}
+) => {
   return {
+    reasonsForUseOfForce: whenPresent(reasonsForUseOfForce.reasons, reasons =>
+      reasons.map(value => toLabel(UofReasons, value)).join(', ')
+    ),
+    primaryReason: whenPresent(reasonsForUseOfForce.primaryReason, value => toLabel(UofReasons, value)),
```

Comment:
two new fields which may or may not be present when rendering reports

Reply:
ok